test/extended/operators/operators.go

```diff
@@ -259,6 +259,5 @@ func isKubeAPIUpgradableNoUpgradeCondition(cond configv1.ClusterOperatorStatusCo
 	return (cond.Reason == "FeatureGates_RestrictedFeatureGates_TechPreviewNoUpgrade" ||
 		cond.Reason == "FeatureGates_RestrictedFeatureGates_CustomNoUpgrade") &&
 		cond.Status == "False" &&
-		cond.Type == "Upgradeable" &&
-		cond.Message == "FeatureGatesUpgradeable: \"TechPreviewNoUpgrade\" and \"CustomNoUpgrades\" do not allow updates"
```

Comment:
the point of the string selection is to keep us from allowing techpreview clusters to go upgradeable=false for other reasons.  Looks like the string is incorrect?

Reply:
yes, the point of this conditional is to fail-fast-ish in cases where we are `Upgradeable=False` for unexpected reasons (it isn't limited to TechPreview).  #26449, of which this is a robotic pick, says "you know what, who cares about the `Message`?  (`status`, `type`, `reason`) is enough of a tuple for robots to be confident that this is an allowed reason".  If you prefer to also watch over the operator's shoulder and enforce a specific message, I'm... grudgingly ok with that, but can you litigate it in `master` and then continue with the robotic backports to 4.9?

Reply:
> the point of the string selection is to keep us from allowing techpreview clusters to go upgradeable=false for other reasons

doesn't checking the `Reason` ensure this, in a more robust(less fragile/subject to change) way?

Reply:
> 

likely not more robust, but robust enough.